http/action/src/main/java/io/knotx/databridge/http/action/HttpAction.java

```diff
@@ -249,18 +270,35 @@ private void updateResponseMetadata(EndpointResponse response, ActionPayload pay
   }
 
   private ActionPayload handleErrorResponse(ActionRequest request, String statusCode,
-      String statusMessage) {
+      String statusMessage, ActionLogger actionLogger) {
+    ActionPayload payload = ActionPayload.error(request, statusCode, statusMessage);
+    Optional.ofNullable(payload.getResponse())
```

Comment:
I think that this invocation `payload.getResponse()` can not produce null. You can skip this null check.

Reply:
fixed